MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.PasswordBox.xaml

```diff
@@ -406,6 +408,13 @@
     <Setter Property="wpf:HintAssist.Foreground" Value="{DynamicResource PrimaryHueMidBrush}" />
     <Setter Property="wpf:TextFieldAssist.TextBoxViewMargin" Value="{x:Static wpf:Constants.DefaultTextBoxViewMargin}" />
     <Setter Property="wpf:TextFieldAssist.UnderlineBrush" Value="{DynamicResource PrimaryHueMidBrush}" />
+    <Setter Property="wpf:StylizedBehaviors.Behaviors">
```

Comment:
@Keboo This addition here has me a little concerned. Basically we now always apply the behavior on a `PasswordBox` in order to be able to support bindings via the `PasswordBoxAssist.Password` attached property. However, in doing so the consequence is that if you're using MDIX and just want a `PasswordBox` **without** using a binding, then the clear text password is still exposed in the value of that attached property (because the behavior wires it up). So there is really no way to opt-out of it.

Another approach would be not to add it to the "non-reveal" `Style` (NB. it is required in the reveal `Style` for the reveal `TextBox` to sync), and then let the user opt-in for it. Either by setting yet another attached property, or by directly adding the behavior; the latter would then require the behavior to be public.